### PR TITLE
166006697 Tooltip Styling

### PR DESCRIPTION
**Pull request by ILysenkoDA**
#### Related issues
Fixes https://www.pivotaltracker.com/story/show/166006697

#### Description
Icon: 24x24
Padding: 5px 

#### Screenshots (if applicable)
image

### CODE REVIEW

File: assets/css/main.css [hunk 1]
```diff
@@ -4775,7 +4775,10 @@ input.field--reset:active {
 
 .tooltip-closeBtn {
   float: right;
-  padding: 4px; }
+  padding: 5px !important; }
```
Comment (v-gtsolov) [inline assets/css/main.css]: is there a way to escape the `!important` part?

Comment (ILysenkoDA) [reply assets/css/main.css]: @v-gtsolov The button icon prop overwriting padding of the icon

Comment (v-gtsolov) [reply assets/css/main.css]: can we make stronger selector for it than, and remove the `!important` part

Comment (hsbacot) [reply assets/css/main.css]: This will get revisited by us with design and may be reverted back to 20px, however this is good for now

**Pull request merged**